src/components/tooltip/tooltip.st.css

```diff
@@ -0,0 +1,82 @@
+@namespace "Tooltip";
+:import {
+    -st-from: "../../project.st.css";
+    -st-named:
+        BK1;
+}
+:vars {
+    tooltipOffset: 5px;
+    tooltipColor: value(BK1);
+}
+
+.root {
+    -st-states: open;
+    position: absolute;
+    display: none;
+    z-index: 9;
+}
+
+.root:open {
+    display: block;
+}
+
+.tooltip  {
+    text-align: center;
+    background: value(tooltipColor);
+    padding: 12px 25px;
+    color: white;
```

Comment:
use appropriate var so we don't lost this in future

Reply:
`+`